api/v1/announcements/PKPAnnouncementHandler.php

```diff
@@ -92,6 +95,14 @@ public function __construct()
      */
     public function authorize($request, &$args, $roleAssignments)
     {
+        if (!Config::getVar('features', 'site_announcements') && !$request->getContext()) {
```

Comment:
Just to check that we're aligned on the use of feature flags, this would be removed from the `main` forward-port, correct?

Reply:
Yep, I added the feature flag in a separate commit so I can just not cherry-pick that commit over.